site/docs/how-to/review.md

```diff
@@ -0,0 +1,79 @@
+---
+sidebar_position: 5
+slug: '/review'
+---
+
+# Peer review infrastructure changes
+
+Eng teams move over from using cloud UIs to IaC because cloud infrastructure changes can lead to outages more often than business logic changes. As a result, it is considered a DevOps best practice to code review infrastructure changes using IaC tools and show the `preview` of the resulting change in pull requests to raise the quality bar and reduce the possibility of mistakes. We do not recommend that cloud infrastructure changes done with IaSQL be code-reviewed using migration systems since the IaSQL schema is defined by [modules](../concepts/module.md) that can be upgraded and changed over time. Furthermore, we recommend complex, or delicate, infrastructure changes should be treated as transactions in IaSQL. [IaSQL transactions](../concepts/transaction.md) are a bit different from normal DB transactions in that they are multiuser and can be previewed with the [`iasql_preview`](../modules/builtin/iasql_functions.md) function for the changes that the ongoing transaction would create in the cloud, similar to Pulumi’s `preview` or Terraform’s `plan`.
```

Comment:
Restructuring things a bit to be clearer, and breaking up this paragraph into a few for better emphasis of the various points.

```suggestion
Engineering teams move from using cloud UIs to IaC because cloud infrastructure changes can lead to outages more often than business logic changes. As a result, it is a DevOps best practice to code review infrastructure changes using IaC tools, showing the `preview` of the resulting change in pull requests to raise the quality bar and reduce the possibility of mistakes.

The standard practice for code review of database changes is to write the changes via a [migration system](https://en.wikipedia.org/wiki/Schema_migration) that are executed after approval. We do not recommend that cloud infrastructure changes done with IaSQL follow this pattern for two reason: first, these migration tools often do not support data changes, only schema changes, and the IaSQL schema is defined by [modules](../concepts/module.md) that can be upgraded and changed over time, and second, you lose all of the feedback the database can give you on whether or not your SQL statements are valid and what impact they will have on your cloud infrastructure.

Instead, we recommend complex, or delicate, infrastructure changes should be treated as transactions in IaSQL. [IaSQL transactions](../concepts/transaction.md) are a bit different from normal DB transactions in that they lock all database changes from propagating into the cloud but do not lock the changes from being viewed by other database connections, making them "multiuser", so others can inspect the changes, and can preview what they would do with the [`iasql_preview`](../modules/builtin/iasql_functions.md) function, similar to Pulumi’s `preview` or Terraform’s `plan`.
```